web/src/main/java/org/springframework/security/web/server/authentication/ott/GenerateOneTimeTokenWebFilter.java

```diff
@@ -75,4 +74,15 @@ public void setRequestMatcher(ServerWebExchangeMatcher matcher) {
 		this.matcher = matcher;
 	}
 
+	/**
+	 * Use the given {@link ServerGenerateOneTimeTokenRequestResolver} to resolve
+	 * {@link GenerateOneTimeTokenRequest}..
```

Comment:
Please remove the stray period. Also please state the default value (`DefaultServerGenerateOneTimeTokenRequestResolver`)